evalplus/provider/gptqmodel.py

```diff
@@ -0,0 +1,87 @@
+from typing import List
+
+import torch
+from transformers import AutoTokenizer
+
+try:
+    from gptqmodel import GPTQModel, get_backend
+except ModuleNotFoundError as exception:
+    raise type(exception)(
+        "Tried to load gptqmodel, but gptqmodel is not installed ",
+        "please install gptqmodel via `pip install gptqmodel --no-build-isolation`",
+    )
```

Comment:
Maybe it's safe to assume `gptqmodel` is already installed given that this file is only imported when user specifies `gptqmodel` as the backend provider.

Reply:
@ganler This give better user-facing error than  an import error stack-trace with error resolution. Just in case users invoke gptqmodel path without actually installing gptqmodel.